.travis.yml

```diff
@@ -1,23 +1,27 @@
 # see http://about.travis-ci.org/docs/user/languages/php/ for more hints
 language: php
+
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
 matrix:
-    allow_failures:
-        - php: hhvm
+  fast_finish: true
+  allow_failures:
+    - php: 7.0
+    - php: hhvm
 
 before_script:
-  - curl -s http://getcomposer.org/installer | php
-  - php composer.phar install --dev --no-interaction
+  - composer self-update
+  - composer install --prefer-source --no-interaction
```

Comment:
@Soullivaneuh Added.

Btw just a tip: link in your previous message is outdated. To prevent that, just pres "Y" on the url or picked lines. It will change link to [absolute one](https://github.com/Soullivaneuh/IsoCodesValidator/blob/597486d1eb96717ffd3408d3ed7864e07e6476fd/.travis.yml#L39-L41)